sources/packages/web/src/assets/css/global-style-variables.scss

```diff
@@ -18,7 +18,7 @@ $banner-background-info: #f0eef9;
 /** Background and font colors - Figma **/
 $common-border-color: #e8e8e8;
 $color-white: #ffffff;
-$background-read-only: #cfd4db;
+$read-only-background: #e9ecef;
```

Comment:
do we need to update vuetify theme `readonly`?
![image](https://user-images.githubusercontent.com/77353155/200083726-e9316e1f-39cd-47c6-a125-019efda7a9f8.png)
so, these are used by the vuetify components

Reply:
@ann-aot  Oh yes we should - thank you for noting that! I changed it now.